tests/unit/Core/Framework/DataAbstractionLayer/Field/Flag/AssociationDescriptionTest.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Core\Framework\DataAbstractionLayer\Field\Flag;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\AssociationDescription;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[CoversClass(AssociationDescription::class)]
+#[Package('framework')]
+class AssociationDescriptionTest extends TestCase
```

Comment:
I don't like such tests, as they bring not really any value. for me it would be okay to add the code coverage ignore annotation to the class